### DISCUSSION TITLE
Being able to undo `close`s

### DISCUSSION THREAD

**Discussion by omid** [zellij-org/zellij#1007] [Ideas]
When a user closes a tab or pane, there can be a timeout of `n` seconds, between hiding them and closing them.

- In the meanwhile, user can undo the action. (reopen the tab or pane)
- `n` is be configurable in the settings (even can be 0)

**Comment by MisieEugene** [comment]
This feature will be very useful

**Comment by cristiand391** [comment]
Not sure if a timeout or "save last X panes/tab" option is better, but I would definitely make it work like session resurrection where it stops resurrected commands and put them under a `Press ENTER to run...` banner instead of just "hiding" a pane/tab.

https://zellij.dev/documentation/session-resurrection#what-is-resurrected-and-how-to-configure
